xmodule/split_test_block.py

```diff
@@ -172,10 +173,14 @@ def child_block(self):
     def child(self):
         """
         Return the user bound child block for the partition or None.
+
+        We are using try/except block here because we ran into issues with
+        CachingDescriptorSystem has no attribute when we get an icon for the split_test xblock.
```

Comment:
The comment should be changed specifying the technical issue
and note just - `we get an icon for the split_test xblock`

e.g.
Handles the `AttributeError` exception that may occur when attempting to retrieve the `split_test` xBlock information within the CMS.